requirements-test-libraries.txt

```diff
@@ -96,6 +96,7 @@ pyqtgraph==0.13.7; python_version >= "3.9"
 pyusb==1.3.1; python_version >= "3.9"
 pyviz-comms==3.0.6
 pyvjoy==1.0.1; sys_platform == "win32"
+pynng==0.8.1; python_version >= "3.7"
```

Comment:
We support only python >= 3.8 here, so `; python_version >= "3.7"` is redundant.

Reply:
Fixed in  b1b72fa